docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -335,13 +335,7 @@ Now, if you navigate to your robot's **Control** tab, you should be able to cont
 
 ![Creation of a pump motor in the Viam app config builder.](/tutorials/plant-watering-pi/pump-motor-control.png)
 
-{{% alert title="Tip" color="tip" %}}
-
-Now that you have set up your robot and are able to control your motor, you can put the suction tube of your pump into the water cup, and the output tube into the plant!
-
-{{% /alert %}}
-
-### Configure the ADC as a module from the registry
+#### Configure the ADC as a module from the registry
```

Comment:
Moved down one header block to make it part of "setting up your robot in the Viam app", was not clear before